web-ui/src/components/split-button/SplitButton.jsx

```diff
@@ -57,7 +57,7 @@ const SplitButton = ({toggleOnSelect=false, onClick, options}) => {
             <ArrowDropDownIcon />
           </Button>
         </ButtonGroup>
-        <Popper open={open} anchorEl={anchorRef.current} role={undefined} placement="bottom-end" transition disablePortal>
+        <Popper open={open} anchorEl={anchorRef.current} role={undefined} placement="bottom-end" transition >
```

Comment:
What's the reason for this change?

Reply:
it the disableProtal is in there the split button goes behind the picture of the person below and you can't see it

Reply:
ok...does this impact where the popper opens?

Reply:
nope, just the way it shows up